packages/trader/src/AppV2/Components/Guide/Description/video-fragment.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Localize } from '@deriv/translations';
+import { Loading } from '@deriv/components';
+import { Text } from '@deriv-com/quill-ui';
+import { useDevice } from '@deriv-com/ui';
+import { getUrlBase } from '@deriv/shared';
+
+type TVideoFragment = {
+    contract_type: string;
+};
+const VideoFragment = ({ contract_type }: TVideoFragment) => {
+    const [is_loading, setIsLoading] = React.useState(true);
+
+    const { isMobile } = useDevice();
+
+    // memoize file paths for videos and open the modal only after we get them
```

Comment:
The logic was taken from existing file packages/trader/src/Modules/Contract/Components/AccumulatorsStats/accumulators-stats-manual-modal.tsx